### PR TITLE
ci(screener): integrating SauceLabs account with Screener for finer-grained browser control

### PR DESCRIPTION
**Pull request by eriklharper**
**Related Issue:** n/a

## Summary

This pr adds Sauce Labs credentials to Screener with the newly-created, dedicated account.  This should allow us to specify more recent Chrome versions.

The Chrome version now appears on each parallelized run in the Screener UI under "Show Logs"

image

### CODE REVIEW

**Review by jcfranco — APPROVED** [review]
🎉

File: screener.config.js [hunk 1]
```diff
@@ -8,11 +8,17 @@ module.exports = {
   baseBranch: "master",
   browsers: [
     {
-      browserName: "chrome"
+      browserName: "chrome",
+      version: "87.0"
```
Comment (driskull) [inline screener.config.js:12]: don't we want 89 or greater?

Comment (eriklharper) [reply screener.config.js:12]: We do, but 87 is currently the highest supported Chrome version according to: https://docs.saucelabs.com/visual/component-testing/integrations/sauce-labs/#supported-browsers

Comment (eriklharper) [reply screener.config.js:12]: Also, they don't have a `latest` option either, but they did submit that idea as product feedback.

Comment (driskull) [reply screener.config.js:12]: Didn't they say that doc was outdated in the email?

Comment (driskull) [reply screener.config.js:12]: >Please disregard the supported browsers section, as this appears to be outdated. I have reached out to our documentation team about this discrepancy.

Comment (eriklharper) [reply screener.config.js:12]: Wow, I totally overlooked that.  I reached out to them to ask them what is going on, so hopefully they can provide me the proper config.

Comment (driskull) [reply screener.config.js:12]: Can you change it to 89?

Comment (benelan) [reply screener.config.js:12]: Once we get it working on 87 we can bump it to see if it works on 89. Right now we just need to get it all set up correctly.

**Pull request merged**